src/module/auth/login-form.tsx

```diff
@@ -1,19 +1,20 @@
-"use client";
```

Comment:
looks like we are using different formatter options, maybe add a prettier config so that everything can be enforced to either `"` or `'`?

I don't have a preference, feel free to choose at your discretion (or checkout what `beequant_app` is doing)

Reply:
From what I can tell, BeeQuant uses the `"` format uniformly. I seem to have gotten that way because I cloned the `beequant_app`

Reply:
uhhh, okay? so.... `"` yeah? you wanna change your settings?

Reply:
surely!